lib/MiniDroneBtAdapter.js

```diff
@@ -403,7 +403,10 @@ class MiniDroneBtAdapter extends EventEmitter {
         const manufacturer = peripheral.advertisement.manufacturerData;
         const matchesFilter = localName === this.options.droneFilter;
 
-        const localNameMatch = matchesFilter || DRONE_PREFIXES.some((prefix) => localName.indexOf(prefix) >= 0);
```

Comment:
I don't have a Mambo Bebop, but I think there might be a more elegant way to fix this. What is the values of the `peripheral.advertisement` object at this line? Does `peripheral.advertisement.localName` start with `Mambo_`? Or something else?

If it's something else you can probable fix this by just adding it to `DRONE_PREFIXES` at the top of this class